code/_onclick/hud/alert.dm

```diff
@@ -19,9 +19,8 @@
 	if(!category)
 		return
 
-	var/obj/screen/alert/alert
-	if(alerts[category])
-		alert = alerts[category]
+	var/obj/screen/alert/alert = LAZYACCESS(alerts, category)
```

Comment:
be sure to test alerts thoroughly and look through the code to ensure that things are consistent with use of this new system.

Reply:
I tested with human, cat and other various mobs and I didn't see any problems so far